tests/functional/centos/7/ooo-collocation/hosts

```diff
@@ -19,8 +19,8 @@ all:
     cephfs_data: manila_data
     cephfs_metadata: manila_metadata
     cephfs_pools:
-    - {name: manila_data, pgs: 8}
-    - {name: manila_metadata, pgs: 8}
+    - {name: manila_data, pgs: 8, size: ""}
```

Comment:
this might break OSP deployment.
@gfidente can you take a look, please?

Reply:
adding a default as suggested [here](https://github.com/ceph/ceph-ansible/pull/3145#discussion_r225091047) should provide backward compatibility.

Reply:
@rishabh-d-dave I  think we should leave that file as it is so the CI would cover the backward compatibility

Reply:
If I am recalling it correctly, not adding the size attribute here leads to an `KeyError`.

Reply:
@rishabh-d-dave if you add that `default("")` value when `size` is called, it shouldn't.
in THT, that attribute won't be added anyway.
When bringing such a change we owe to keep backward compatibility, this is why I want this covered/validated by the CI.

Reply:
> @rishabh-d-dave if you add that default("") value when size is called, it shouldn't.

Yes, you are right.